### PR TITLE
Update the RNA BP fields in the bioinfo tab

### PR DESCRIPTION
**Pull request by ewels**
Replaced the previous RNA BP fields with the new fields for the bioinfo tab. Tested locally, seems to work ok. Not made any changes with the bioinfo-tab stuff before, so would appreciate someone looking over it :)

Ignore the old commits here, they were just writing some code and then removing it again (ended up being written a different way by Denis).

### CODE REVIEW

**Comment by ewels** [comment]
Running on stage: https://genomics-status-stage.scilifelab.se/bioinfo/P6759

**Comment by ewels** [comment]
image

**Comment by vezzi** [comment]
in the stage version there are 4 fields but they are all "unique mapping rate"

File: run_dir/design/bioinfo_tab/run_lane_sample_view.html [hunk 1]
```diff
@@ -38,8 +38,10 @@
       <th class="bioinfo-status-th bioinfo-status-qc fastq_screen">fastq_screen</th>
       <th class="bioinfo-status-th bioinfo-status-qc blast_wrapper">blast_wrapper</th>
       {% if application =='rnaseq' %}
-        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_sequenceduplication">Sequence<br>duplication</th>
-        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_uniquemappingrate">Unique mapping rate</th>
+        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_alignment">Unique mapping rate</th>
+        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_rrna">Unique mapping rate</th>
+        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_strandspecificity">Unique mapping rate</th>
+        <th class="bioinfo-status-th bioinfo-status-bp rnaseq_duplication">Unique mapping rate</th>
```
Comment (vezzi) [inline run_dir/design/bioinfo_tab/run_lane_sample_view.html]: yep here

Comment (ewels) [inline run_dir/design/bioinfo_tab/run_lane_sample_view.html]: Oops! Good spot, sorry.

**Comment by ewels** [comment]
Ok, fixed (also updated the autoreload statements so that I didn't have to manually reboot tools-dev).

**Comment by ewels** [comment]
Ready to be merged and deployed to production when the new RNA pipeline goes into prod.

**Comment by ewels** [comment]
New field added for `MT rRNA`.

**Comment by ewels** [comment]
Seems to be working fine:

image

**Comment by vezzi** [comment]
Great.. merging in preparation for new pipeline going in production 18th of April 2017

**Comment by vezzi** [comment]
@ewels we can already deploy it in production. inform jun once it is online thanks

**Comment by ewels** [comment]
Deployed!

**Pull request merged**